mlir/lib/Dialect/Tensor/IR/TensorTilingInterfaceImpl.cpp

```diff
@@ -246,6 +246,97 @@ struct PackOpTiling
       return failure();
     return tilingResult.value();
   }
+
+  /// Method to return the position of iteration domain tile computed by the
+  /// tiled operation. In current `tensor.pack` context, the `resultOffsets` and
+  /// `resultSizes` only cover outer dimensions.
+  LogicalResult getIterationDomainTileFromOperandTile(
+      Operation *op, OpBuilder &b, unsigned operandNumber,
```

Comment:
You should be checking the operand number somewhere?

Reply:
Ok, I have added check. BTW, it is not used in fact.

Reply:
You are checking that this is in fact the 0-th operand (and then hard-wiring the fact that it is the zero-th operand later. SO you are in fact using it :)

Reply:
Ok, I see.